Randomizer.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+
+def randomly(list_of_students):

```

Comment:
Function names should be verb, not adverb

Reply:
Done:)